test/functional/services/lib/web_element_wrapper/web_element_wrapper.ts

```diff
@@ -160,7 +160,7 @@ export class WebElementWrapper {
   async clearValueWithKeyboard(options: TypeOptions = { charByChar: false }): Promise<void> {
     if (options.charByChar === true) {
       const value = await this.getAttribute('value');
-      for (let i = 1; i <= value.length; i++) {
+      for (let i = 0; i <= value.length; i++) {
```

Comment:
in this case you send `Backspace` event length + 1 times. Do you really need an additional one?

Reply:
yeah, tested locally a lot of times, In this case(`i = 1`) it was 1 letter. Example word `test` after executing this method it was `t`